src/backend/db/categories.js

```diff
@@ -8,20 +8,31 @@ import { v4 as uuid } from "uuid";
 export const categories = [
   {
     _id: uuid(),
-    categoryName: "fiction",
-    description:
-      "literature in the form of prose, especially novels, that describes imaginary events and people",
+    categoryName: "Tshirts",
+    tagLine: "WEAR YOUR BEST MOMENTS",
+    imgUrl: "https://cdn.shopify.com/s/files/1/0220/4008/4552/products/staple_6pack_1024x1024.jpg?v=1640718788"
   },
   {
     _id: uuid(),
-    categoryName: "non-fiction",
-    description:
-      "Non-fiction is writing that gives information or describes real events, rather than telling a story.",
+    categoryName: "Casual Pants",
+    tagLine: "Simple Yet Elegant",
+    imgUrl: "https://ae01.alicdn.com/kf/H0218744e90d1406288714f11429ea24ex/2020-New-Men-s-Cotton-Linen-Pants-Male-Summer-Breathable-Solid-Color-Linen-Trousers-Fitness-Streetwear.jpg_Q90.jpg_.webp"
```

Comment:
Looks like the url for the casual pants image is not working, couldn't see the image.

Reply:
It is working properly: 
![image](https://user-images.githubusercontent.com/76832165/159559583-4b4613d5-dbe1-4cdc-a476-f7aa021cb3b1.png)